src/bin/sol-fbp-runner/web-inspector.c

```diff
@@ -36,6 +36,47 @@ static struct sol_http_progressive_response *events_response;
 
 static struct timespec start;
 
+
+static int
+web_inspector_send_sse_data(struct sol_http_progressive_response *client, struct sol_buffer *buf)
+{
+    int r;
+    struct sol_blob *blob;
+    static struct sol_blob *sse_prefix = NULL;
+    static struct sol_blob *sse_suffix = NULL;
+
+    if (!sse_prefix) {
+        sse_prefix = sol_blob_new(SOL_BLOB_TYPE_NO_FREE, NULL, "data: ", strlen("data: "));
```

Comment:
this should be a global `static struct sol_blob sse_prefix`, no need to call `sol_blob_new()`. However I see why you did it and we should improve that: `SOL_BLOB_TYPE_NO_FREE` should be renamed to `SOL_BLOB_TYPE_NO_FREE_DATA`, and `SOL_BLOB_TYPE_NO_FREE` should be an empty free function in `struct sol_blob_type`, this is because if `type->free == NULL`, then it will free the blob structure, which is not wanted.